model/model.go

```diff
@@ -182,6 +182,10 @@ func (si *SystemInstall) Validate() error {
 		}
 	}
 
+	if si.Timezone == nil {
```

Comment:
why are we considering timezone mandatory, if it's not provided then we should fallback to a default one - could be defined with the default configuration.

Reply:
you do what I'm suggesting with keyboard, just replicate it here maybe.

Reply:
If you look around line 250, we do default to the UTC if one is not defined. That check is a just a last ditch check or if code changes in the future to avoid a runtime nil error.